src/main/kotlin/com/buildit/bookit/v1/booking/BookingRepository.kt

```diff
@@ -14,63 +13,48 @@ import java.util.concurrent.atomic.AtomicInteger
 val local: ZoneId = ZoneId.systemDefault()
 val utc: ZoneId = ZoneId.of("UTC") // this should be replaced with that of the bookable
 
-fun mapFromResultSet(rs: ResultSet): Booking
```

Comment:
I still think this might still be useful since jdbc template still needs a mapper to an item and I can see it being reused